### PR TITLE
Sonar issues fixed.

### PR DESCRIPTION
**Pull request by TARGETrus**

### CODE REVIEW

File: src/applications/basic_analytics/vspp/diagnostic_server_driver.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class VsppDiagnosticServer(BasicAnalyticsProcessor):
 
     def _process_pipeline(self, read_stream):
         mapping = {
-            ".*error got \d*, expected \d*, packet \d*.*":
+            ".*error got \\d*, expected \\d*, packet \\d*.*":
```
Comment (halfwhite) [inline src/applications/basic_analytics/vspp/diagnostic_server_driver.py]: Use """

File: src/common/basic_analytics/aggregations.py [hunk 2]
```diff
@@ -42,12 +42,12 @@ def __get_use_udf(self):
...
-            if isinstance(column, basestring):
-                if self.__get_use_udf():
-                    column = self.__quote_chars_udf(column)
-                return regexp_replace(regexp_replace(column, "\\.",  "%2E"), "\\s+", "%20")
-            else:
-                return regexp_replace(column, "\\s+", "%20")
+        if isinstance(column, basestring):
+            if self.__get_use_udf():
+                column = self.__quote_chars_udf(column)
+            return regexp_replace(regexp_replace(column, "\\.",  "%2E"), "\\s+", "%20")
```
Comment (halfwhite) [inline src/common/basic_analytics/aggregations.py]: use """

File: src/common/basic_analytics/aggregations.py [hunk 3]
```diff
@@ -42,12 +42,12 @@ def __get_use_udf(self):
...
+        else:
+            return regexp_replace(column, "\\s+", "%20")
```
Comment (halfwhite) [inline src/common/basic_analytics/aggregations.py]: use """

File: src/applications/basic_analytics/vspp/diagnostic_server_driver.py (hunk 1 above)
Comment (vunder-kind) [inline src/applications/basic_analytics/vspp/diagnostic_server_driver.py]: Just make this string raw.

File: src/common/basic_analytics/aggregations.py (hunk 2 above)
Comment (vunder-kind) [inline src/common/basic_analytics/aggregations.py]: Also remove double slash and use raw string

**Pull request merged**